src/state/apis/snapshot/snapshot.ts

```diff
@@ -142,5 +147,62 @@ export const snapshotApi = createApi({
         }
       },
     }),
+    getProposals: build.query<Proposal[], void>({
+      queryFn: async (_, { dispatch }) => {
+        const query = `
+          query Proposals {
+            proposals(
+              first: 20,
+              skip: 0,
+              where: {
+                space: "${SNAPSHOT_SPACE}",
+              },
+              orderBy: "created",
+              orderDirection: desc
+            ) {
+              id
+              author
+              created
+              network
+              symbol
+              type
+              title
+              body
+              discussion
+              choices
+              labels
+              start
+              end
+              quorum
+              quorumType
+              privacy
+              snapshot
+              state
+              link
+              app
+              scores
+              scores_state
+              scores_total
+              scores_updated
+              votes
+              flagged
```

Comment:
Suggestion: actually leverage GraphQL to request only the fields we want 
```suggestion
	id
	title
	body
	choices
	scores
	scores_total
	link
```

And cleanup zod schema without the additional ones

Reply:
Fixed in [2b38bd6](https://github.com/shapeshift/web/pull/7952/commits/2b38bd691a2b53a6c5fc056cd4fe189082ca3806) , minus `id` and `state` fields that are also used